repos/fdbt-site/tests/pages/api/multiTapsPricing.test.ts

```diff
@@ -0,0 +1,97 @@
+import { getMockRequestAndResponse } from '../../testData/mockData';
+import * as apiUtils from '../../../src/utils/apiUtils';
+import { MULTI_TAPS_PRICING_ATTRIBUTE, NUMBER_OF_TAPS_ATTRIBUTE } from '../../../src/constants/attributes';
+import * as sessions from '../../../src/utils/sessions';
+import multiTapsPricing from '../../../src/pages/api/multiTapsPricing';
+
+describe('multiTapsPricing', () => {
+    let writeHeadMock: jest.Mock;
+    const updateSessionAttributeSpy = jest.spyOn(sessions, 'updateSessionAttribute');
+
+    beforeEach(() => {
+        writeHeadMock = jest.fn();
+    });
+
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const cases: any[] = [
+        [
+            {
+                multiTapPriceInput0: '-1',
+            },
+            { Location: '/multiTapsPricing' },
+        ],
+
+        [
+            {
+                multiTapPriceInput0: '',
+            },
+            { Location: '/multiTapsPricing' },
+            'period',
```

Comment:
Is there meant to be this extra member of the array? It looks like we only use the first two.